src/community/roadmap/index.md

```diff
@@ -23,32 +23,38 @@ We recently:
 - released the [Summary card](https://github.com/alphagov/govuk-design-system-backlog/issues/210) variant
 - made it easier to find information about [contributing and the community](/community/)
 - improved [how contributions are prioritised](https://design-system.service.gov.uk/community/upcoming-components-patterns/)
-- added functionality for browsers to ‘find in page’ within the Accordion component`
+- added functionality for browsers to ‘find in page’ within the Accordion component
 - added support for ‘source maps’ for precompiled GOV.UK Frontend Javascript files
+- iterated the [guidance on alt text for images](/styles/images/#alt-text)
 
 ## Working on now
 
-We're working to:
+We're working on GOV.UK Frontend v5.0 which includes:
 
-- host community co-design work to [turn task-list into a component](/patterns/task-list-pages/)
-- host community co-design work to make maps better
-- release the [Hide this page](https://github.com/alphagov/govuk-design-system-backlog/issues/213) component
-- improve how [JavaScript works in GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/issues/1389)
-- iterate on alt text guidance for the images style page
-- make it easier for teams to share their findings from user research
+- [updating our browser support](https://github.com/alphagov/govuk-frontend/issues/2621), including removing support for Internet Explorer 8-10 and reducing support for Internet Explorer 11
```

Comment:
I’d prefer an en dash (`–`) instead of a hyphen, but it’s not a blocking change.

Reply:
Should we do similar to GOV.UK's [recommendation for date ranges](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates) and use 'to' instead?

> …including removing support for Internet Explorer 8 to 10